src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,15 @@
+export const Tab = ({ tab, onTabSelected, activeTabId }) => {
+  return (
+    <li data-cy="Tab" className={tab.id === activeTabId.id ? 'is-active' : ''}>
```

Comment:
The comparison `tab.id === activeTabId.id` is incorrect. Since `activeTabId` should be a string representing the ID, the comparison should be `tab.id === activeTabId`.